### PR TITLE
Update docs to use for v3.0.3

### PR DESCRIPTION
**Pull request by caseydavenport**
## Description

Update docs to use Calico v3.0.3

### CODE REVIEW

File: _data/versions.yml [hunk 1]
```diff
@@ -1,4 +1,74 @@
...
+      url: https://github.com/projectcalico/typha/releases/tag/v0.6.2
+    calicoctl:
+      version: v2.0.1
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v2.0.1
+      download_url: https://github.com/projectcalico/calicoctl/releases/download/v2.0.1/calicoctl
+    calico/node:
+      version: v3.0.3
+      url: https://github.com/projectcalico/calico/releases/tag/v3.0.3
+    calico/cni:
+      version: v2.0.1
```
Comment (tmjd) [inline _data/versions.yml]: I think this should be v2.0.2

Comment (caseydavenport) [reply _data/versions.yml]: ugh, it should be, but the artifacts are already shipped with v2.0.1, so for now I think this has to stay.

File: _data/versions.yml [hunk 2]
```diff
@@ -1,4 +1,74 @@
 v3.0:
+- title: v3.0.3
+  note: |
+    23 February 2018
```
Comment (emanic) [inline _data/versions.yml]: 26?

Comment (caseydavenport) [reply _data/versions.yml]: Nope, the release went out on the 23rd: https://github.com/projectcalico/calico/releases/tag/v3.0.3

**Pull request merged**